packages/jest-utils/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@empathyco/x-jest-utils",
```

Comment:
If you don't mind, can you follow the same structure as in the other packages?

Reply:
Sure!